pixano/features/schemas/annotations/text_generation.py

```diff
@@ -0,0 +1,116 @@
+# =====================================
+# Copyright: CEA-LIST/DIASI/SIALV/LVA
+# Author : pixano@cea.fr
+# License: CECILL-C
+# =====================================
+
+from datetime import datetime
+from typing import Literal
+
+from pydantic import model_validator
+
+from pixano.utils import issubclass_strict
+
+from ...types.schema_reference import EntityRef, ItemRef, SourceRef, ViewRef
+from ..registry import _register_schema_internal
+from .annotation import Annotation
+
+
+@_register_schema_internal
+class Message(Annotation):
+    """Textual exchange in a question/answer conversation
+    for image or text description and inforation extraction.
+
+    Attributes:
+        number: message number to associate diffrent ANSWER messages to a QUESTION.
+        user: identify who is the author of the message (eg a human, a model, the ground truth, etc).
+        content: actual text of the message.
+        timestamp: creation date of the message.
```

Comment:
there is already created_at & updated_at  timestamp in BaseSchema

Reply:
Good remark. @juliendenize  , do you think that the timestamp is still necessary in the schema ?

Reply:
Yes to keep track of timestamp outside of Pixano (created is for Pixano) we could change later if we find there is no usecase for this